### ISSUE TITLE
WorkflowExecutionInfo.task_queue is blank in DescribeWorkflowExecutionResponse

### ISSUE THREAD

**Issue by nagl-stripe** [temporalio/temporal#2374]
## Expected Behavior
[DescribeWorkflowExecutionResponse](https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/request_response.proto#L706).workflow_execution_info.[task_queue](https://github.com/temporalio/api/blob/master/temporal/api/workflow/v1/message.proto#L57) seems like it should contain the workflow's task queue.

## Actual Behavior
It's blank.

I did notice that DescribeWorkflowExecutionResponse.execution_config.[task_queue](https://github.com/temporalio/api/blob/master/temporal/api/workflow/v1/message.proto#L62) is set. That unblocks us - this bug is extremely low priority.

## Steps to Reproduce the Problem
Describe any workflow, e.g. with tctl.

## Specifications

  - Version: Observed on 1.13, code inspection suggests 1.14 too.
  - Platform: Mysql, but it doesn't look like it matters.

**Comment by alexshtin** [comment]
Yes, this field was missing in response. Thanks for reporting it!
